--- b2/setup.py	2017-03-23 17:38:56.000000000 -0400
+++ b2/setup.py.fix	2017-08-15 13:54:02.845257412 -0400
@@ -101,7 +101,7 @@
 
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=find_packages(exclude=['contrib', 'docs', 'tests']),
+    packages=find_packages(exclude=['contrib', 'docs', 'test']),
 
     # Alternatively, if you want to distribute just a my_module.py, uncomment
     # this:
